src/aioswitcher/api/__init__.py

```diff
@@ -828,3 +915,50 @@ async def get_shutter_state(self, index: int = 0) -> SwitcherShutterStateRespons
                     "get shutter state request was not successful"
                 ) from ve
         raise RuntimeError("login request was not successful")
+
+    async def set_light(
+        self, command: DeviceState, index: int = 0
+    ) -> SwitcherBaseResponse:
+        """Use for turn on/off light.
+
+        Args:
+            command: use the ``aioswitcher.api.DeviceState`` enum.
+            index: which light to turn on/off, default to 0.
+
+        Returns:
+            An instance of ``SwitcherBaseResponse``.
+
+        """
+        index_packet = get_light_index(self._device_type, index) + 1
```

Comment:
Please add a comment explaining the +1 part.

Reply:
it is done.